lib/routes/eastmoney/gerenzhongxin/trpl.ts

```diff
@@ -0,0 +1,67 @@
+import { Route, ViewType } from '@/types';
+import { parseDate } from '@/utils/parse-date';
+import got from '@/utils/got';
+import timezone from '@/utils/timezone';
+import md5 from '@/utils/md5';
+
+export const route: Route = {
+    path: '/gerenzhongxin/trpl/:uid',
+    categories: ['finance'],
+    view: ViewType.Articles,
+    example: '/gerenzhongxin/trpl/2922094262312522',
+    parameters: { uid: '用户id,即用户主页网址末尾的数字' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['guba.eastmoney.com'],
+        },
+        {
+            source: ['caifuhao.eastmoney.com'],
+        },
+        {
+            source: ['i.eastmoney.com/:uid'],
+            target: '/gerenzhongxin/trpl/:uid',
+        },
+    ],
+    name: '个人中心评论',
+    maintainers: ['AwesomeDog'],
+    handler,
+};
+
+export async function handler(ctx) {
+    const uid = ctx.req.param('uid');
+    const url = `https://i.eastmoney.com/api/guba/myreply?pageindex=1&uid=${uid}&checkauth=true`;
+
+    const response = await got(url, {});
+    const arr = response.data.result.list;
+    const nickname = arr[0].reply_user.user_nickname;
+
+    const items = arr.map((item) => {
+        const descriptionContent = item.reply_text + '<br><br>---主帖内容：---<br><br>' + item.source_post_title;
```

Comment:
Please use `item.source_reply_text || item.source_post_title` for source content.